packages/datatrak-web/src/layout/UserMenu/UserInfo.tsx

```diff
@@ -28,23 +56,52 @@ const LoginLink = styled(AuthLink).attrs({
   border-color: ${props => props.theme.palette.text.primary};
 `;
 
+const useData = () => {
```

Comment:
This is super pedantic, but maybe we could rename this `useUserData` or something to make it more obvious what it is